discovery-infra/test_infra/utils/global_variables/global_variables.py

```diff
@@ -57,12 +62,14 @@ def __post_init__(self):
         self._set("openshift_version", utils.get_openshift_version(allow_default=True, client=client))
 
         for conditions, values in _triggers.items():
-            assert isinstance(conditions, tuple) and isinstance(conditions[0], tuple), f"Key {conditions} must be tuple of tuples"
-            if all(map(lambda condition: self.is_set(condition[0], condition[1]), conditions)):
+            assert isinstance(conditions, tuple) and all(isinstance(condition, tuple)
```

Comment:
this change is just some refactoring / cleaning up, no interesting bug fixes here